README.md

```diff
@@ -8,15 +8,58 @@
 [![codecov.io](https://codecov.io/github/mrc-ide/dust/coverage.svg?branch=master)](https://codecov.io/github/mrc-ide/dust?branch=master)
 <!-- badges: end -->
 
-Fast and simple iteration of stochastic models. Designed to work as a component of a particle filter.
+Stochastic models appear in many domains as they are easy to write out, but hard to analyse without running many realisations of the process. `dust` provides an engine for running stochastic models, taking care of many bookkeeping details such as:
 
-## Background
+* Running stochastic models in parallel on multi-core machines
+* Running models massively in parallel on GPU without writing any low-level code
+* Reducing output by filtering over steps or intermediate outputs
+* Providing useful verbs for working with stochastic models (for initialisation, state setting, parameter updating, and inspection)
+* Combining multiple realisations due to stochastic variability and due to different parameter sets
 
-Stochastic models appear in many domains as they are easy to write out, but hard to analyse without running many realisations of the process. `dust` provides a light interface to run models that are written in C++ in parallel from R. It provides very little functionality aside from a random number generator that is designed to be run in parallel, and is mostly interested in providing an _interface_, with which more powerful tools can be developed.
+## Get started
```

Comment:
I think we should also add the link to the Wellcome Open Research paper somewhere in the README

Reply:
yes, good point. have added that properly, including machine-readable metadata